src/middleware/preAuth.middleware.ts

```diff
@@ -16,7 +16,7 @@ export class PreauthMiddleware implements NestMiddleware {
         .auth()
         .verifyIdToken(token.replace('Bearer ', ''))
         .then(async (decodedToken) => {
-          const { email, uid } = decodedToken;
+          const { email, uid, picture } = decodedToken;
```

Comment:
I don't think picture is any key in decodedToken.
Please correct it.
It is photoUrl

Reply:
If I am correct.

Reply:
Oh. it has the picture property
photoUrl is not there , just checked it again